### PR TITLE
Automation identities

### PR DESCRIPTION
**Pull request by bderveni**
- Surface Automation Identities

### CODE REVIEW

File: Documentation/Identity/Identity_ImplicitClient.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 uid: identityImplicitClient
 ---
 
-# ImplicitClient
+# ImplicitClient (Obsolete)
 
-We suggest using a Authorization Code Client instead of an Implicit Client.
+Implicit Client has been obsoleted. We suggest using a Authorization Code Client instead of an Implicit Client.
```
Comment (osimarkrogers) [inline Documentation/Identity/Identity_ImplicitClient.md]: Change to "an Authorization Code Client"

Comment (osimarkrogers) [inline Documentation/Identity/Identity_ImplicitClient.md]: Also, I believe we should use the term "deprecated" rather than "obsoleted".

Comment (bderveni) [reply Documentation/Identity/Identity_ImplicitClient.md]: Fixed

Comment (bderveni) [reply Documentation/Identity/Identity_ImplicitClient.md]: Fixed.

**Pull request merged**